Kernel/CMakeLists.txt

```diff
@@ -495,6 +495,18 @@ add_compile_options(-fno-exceptions)
 # FIXME: remove -nodefaultlibs after the next toolchain update
 add_compile_options(-nodefaultlibs -nostdlib)
 
+# Auto initialize trivial types on the stack, we use "pattern" as 
+# it's the only option portable across compilers going forward.
+#
+# This is designed to help avoid uninitialized variables bugs and
+# information disclosures coming from the kernel stack. 
```

Comment:
Base/usr/share/man/7/Mitigations could use an entry. Though, looks like we put the commit from the serenity repo in there, so would be a follow-up PR?

Reply:
Yup, that's the plan.